streamalert/alert_processor/outputs/teams.py

```diff
@@ -132,7 +132,8 @@ def _generate_record_section(cls, record):
         This adds the entire record to a section as key/value pairs
 
         Args:
-            record (dict): asd
+            record (dict): The record which caused the Alert to fire
```

Comment:
Could also be `The record that triggered the alert`. A little bit cleaner. Otherwise LGTM

Reply:
Updated to this, much cleaner :)